src/proof_element.rs

```diff
@@ -1,13 +1,19 @@
+use crate::side::Side;
+
 #[derive(Debug, Clone, PartialEq)]
 pub struct ProofElement {
     pub hash: String,
```

Comment:
Here hash can be of the Hash type right?

Reply:
Done!